src/Compilers/VisualBasic/Test/Symbol/SymbolDisplay/SymbolDisplayTests.vb

```diff
@@ -5684,6 +5684,193 @@ class Program
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.NamespaceName,
+                SymbolDisplayPartKind.Operator,
+                SymbolDisplayPartKind.StructName,
+                SymbolDisplayPartKind.Punctuation,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.NamespaceName,
+                SymbolDisplayPartKind.Operator,
+                SymbolDisplayPartKind.StructName)
+
+            Dim ideFormat = New SymbolDisplayFormat(
+                globalNamespaceStyle:=SymbolDisplayGlobalNamespaceStyle.Omitted,
+                genericsOptions:=SymbolDisplayGenericsOptions.IncludeTypeParameters Or SymbolDisplayGenericsOptions.IncludeTypeConstraints,
+                memberOptions:=
+                    SymbolDisplayMemberOptions.IncludeRef Or
+                    SymbolDisplayMemberOptions.IncludeType Or
+                    SymbolDisplayMemberOptions.IncludeParameters Or
+                    SymbolDisplayMemberOptions.IncludeContainingType,
+                kindOptions:=
+                    SymbolDisplayKindOptions.IncludeMemberKeyword,
+                propertyStyle:=
+                    SymbolDisplayPropertyStyle.ShowReadWriteDescriptor,
+                parameterOptions:=
+                    SymbolDisplayParameterOptions.IncludeName Or
+                    SymbolDisplayParameterOptions.IncludeType Or
+                    SymbolDisplayParameterOptions.IncludeParamsRefOut Or
+                    SymbolDisplayParameterOptions.IncludeExtensionThis Or
+                    SymbolDisplayParameterOptions.IncludeDefaultValue Or
+                    SymbolDisplayParameterOptions.IncludeOptionalBrackets,
+                localOptions:=
+                    SymbolDisplayLocalOptions.IncludeRef Or
+                    SymbolDisplayLocalOptions.IncludeType,
+                miscellaneousOptions:=
+                    SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers Or
+                    SymbolDisplayMiscellaneousOptions.UseSpecialTypes Or
+                    SymbolDisplayMiscellaneousOptions.UseErrorTypeSymbolName Or
+                    SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier Or
+                    SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral Or
+                    SymbolDisplayMiscellaneousOptions.CollapseTupleTypes)
+
+            ' When not asking for metadata names, this should show up as an operator.
+            Verify(op.ToDisplayParts(ideFormat),
+                "Operator Integer.=(left As Integer, right As Integer) As Boolean",
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Space,
+                SymbolDisplayPartKind.Keyword,
+                SymbolDisplayPartKind.Operator,
```

Comment:
I guess in VB we happen to use Operator classification for `.` instead of Punctuation? Seems inconsequential, just found it interesting.

Reply:
seems so :)